docs/notebooks/plot_comparisons.ipynb

```diff
@@ -0,0 +1,1118 @@
+{
```

Comment:
<p>Shoot! I'm so happily reading this document that I was not expecting this comment haha!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/695/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Fixed now :)</p>